pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1738,3 +1740,17 @@ func TestRemoveCondition(t *testing.T) {
 		}
 	}
 }
+
+func resourcePath(resource, namespace, name string) string {
+	path := fmt.Sprintf("/apis/%s/%s/", apps.SchemeGroupVersion.Group, apps.SchemeGroupVersion.Version)
```

Comment:
resourcePath being hardcoded to the apps group is unintuitive... I'd expect the testapi resource path helper to be used. if there are issues with that helper, can we resolve those issues instead?

Reply:
> @liggitt do you have concerns about this?

not particularly, now that GKE has apps/v1beta2 enabled. this helper function should be named to make it clear that it is only dealing with apps/v1beta2

Reply:
We're going to punt this to 1.9, since it'll cause problems in updates to HA clusters. See: https://github.com/kubernetes/kubernetes/issues/51049#issuecomment-324784202